lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  def initialize(menu)
+    @select = {}
+    @menu = menu
+    @estimate = 0
+    @client = Twilio::REST::Client.new 'ACe1ded990723482f202fe47c9b5de8a63', '33f4d96523cf8e5308eddf3fa120c615'
```

Comment:
Metrics/LineLength: Line is too long. [111/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)